config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,13 @@
+Sidekiq.configure_server do |config|
+    config.redis = {
+      url: ENV["REDIS_URL"],
+      ssl_params: { verify_mode: OpenSSL::SSL::VERIFY_NONE }
+    }
+  end
+
+  Sidekiq.configure_client do |config|
+    config.redis = {
+        url: ENV["REDIS_URL"],
+        ssl_params: { verify_mode: OpenSSL::SSL::VERIFY_NONE }
+    }
+  end
```

Comment:
I got this to work locally by installing redis via Homebrew and running a redis server locally. Is that similar to how it will work in production and staging?

Reply:
Yup, just added a note to the `README` to track:

https://github.com/turingschool/present/pull/212/commits/75ef0b710093a53fb3b7651f43d9ef01380219ae

Heroku automatically sets the `REDIS_URL` env var when the Redis Add-On is enabled, so we shouldn't need to set any other config.

https://devcenter.heroku.com/articles/connecting-heroku-redis#connecting-from-sidekiq